controllers/topics.js

```diff
@@ -0,0 +1,7 @@
+const { selectAllTopics } = require("../models/topics");
+
+exports.getTopics = (req, res) => {
+  selectAllTopics().then((userTopics) => {
+    res.status(200).send({ userTopics });
+  });
```

Comment:
I'd think about renaming this to be honest - your function is called 'getTopics' but you're not sending topics back - you'll be really upset when you're in your FE project and you can't understand why you can't find `topics` on your backend!

Reply:
I'm not sure what it is sending back, as it's a slug and a description.. is it topic categories?